svc/userservice/internal/app/app.go

```diff
@@ -22,13 +24,15 @@ func NewUserservice(filepath string) (*UserService, error) {
 	if err != nil {
 		return nil, err
 	}
+	cache := newPool(&cfg.Redis)
 
-	repos := newRepos(db)
+	repos := newRepos(db, cache)
```

Comment:
I think for the next development, it would be better to use bridge in (gopkg/bridge), so you can add a new Redis bridge to our project (using bridge design pattern).

Reply:
okaay will do that in next PR for improvement